fw/main.c

```diff
@@ -346,13 +346,15 @@ tfw_ctlfn_state_io(struct ctl_table *ctl, int is_write,
 
 	if (is_write) {
 		char buf[T_SYSCTL_STBUF_LEN];
+		char start[T_SYSCTL_STBUF_LEN] = {"start"};
```

Comment:
Better char start[] = "start"

Reply:
There is main goal of patch to pass arrays with equal explicit size, to satisfy analyzer. But I will remove brackets